README.md

```diff
@@ -1,21 +1,23 @@
-# Dlool
+# Dlool (English)
```

Comment:
The link is an awesome idea! But can we please remove the language in the heading.